src/init.cpp

```diff
@@ -438,9 +438,16 @@ void SetupServerArgs()
 #else
     hidden_args.emplace_back("-upnp");
 #endif
-    gArgs.AddArg("-whitebind=<addr>", "Bind to given address and whitelist peers connecting to it. Use [host]:port notation for IPv6", ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
-    gArgs.AddArg("-whitelist=<IP address or network>", "Whitelist peers connecting from the given IP address (e.g. 1.2.3.4) or CIDR notated network (e.g. 1.2.3.0/24). Can be specified multiple times."
-        " Whitelisted peers cannot be DoS banned", ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
+    std::string permissionsDoc = "";
+    for (const auto& permission : NetPermissions::ToStrings(PF_ALL)) {
+        permissionsDoc = strprintf("%s, %s", permissionsDoc, permission);
+    }
+    permissionsDoc = permissionsDoc.substr(2); // remove starting " ,"
```

Comment:
This code should live in the net_permissions module

Reply:
Why? This is the only place it is needed.